app/src/main/java/org/oppia/app/topic/TopicActivityPresenter.kt

```diff
@@ -8,16 +8,20 @@ import javax.inject.Inject
 
 const val TOPIC_FRAGMENT_TAG = "TopicFragment"
 const val TOPIC_ID_ARGUMENT_KEY = "topic_id"
+const val STORY_ID_ARGUMENT_KEY = "story_id"
 
 /** The presenter for [TopicActivity]. */
 @ActivityScope
 class TopicActivityPresenter @Inject constructor(private val activity: AppCompatActivity) {
-  fun handleOnCreate(topicId: String) {
+
+  fun handleOnCreate(topicId: String, storyId: String) {
     activity.setContentView(R.layout.topic_activity)
     if (getTopicFragment() == null) {
       val topicFragment = TopicFragment()
       val args = Bundle()
       args.putString(TOPIC_ID_ARGUMENT_KEY, topicId)
+      if (!topicId.isEmpty() && !storyId.isEmpty())
```

Comment:
Update this to: `if (topicId.isNotEmpty() && storyId.isNotEmpty())`

Reply:
Done